usr/bin/pop-app-folders

```diff
@@ -192,3 +191,15 @@ fi
 echo "6" > "${CACHE_PATH}"
 
 fi
+
+if [ "${CACHE_VERSION}" -lt "7" ]
+then
+
+echo "Migrating to version 7"
+
+# Default Input volume is too loud. Reduce to 30%
+pactl set-source-volume $(pactl get-default-source) 30%
```

Comment:
This won't help if the input device changes later. Any thoughts on where the default for all devices gets set?

It would be good to use quotes around `$(pactl get-default-source)`.

Reply:
Yes, I know that. From talking to Levi the issue was specifically about the internal mic on the dev one, do we want to expand this issue to all devices?